app/process-helpers.sh

```diff
@@ -75,8 +75,53 @@ run_lua_if_debug() {
   fi
 }
 
+check_if_changed() {
+  directory=$(dirname $1)
+  hash_file=/hashsums/$(basename $1)
+  old_hash=""
+  if [ -f "${hash_file}" ]; then
+    old_hash=$(cat $hash_file)
+    echo -n > $hash_file
+  else
+    export COMPUTE_DIFFS=0
+    touch $hash_file
+  fi
+  find "$directory" -type f -name "*.lua" | sort | xargs shasum > "$hash_file"
+  new_hash=$(cat $hash_file)
+  if [ "$old_hash" == "$new_hash" ]; then
+    echo "0"
+  else
+    echo "1"
+  fi
+}
+
+backup_table() {
+  psql -q -c "DROP TABLE IF EXISTS \"$1_old\";"
+  psql -q -c "ALTER TABLE \"$1\" RENAME TO \"$1_old\";"
```

Comment:
Lass uns einen sprechenderen Namen finden als `_old`. 
- `_diff_source`?
- `_previous_run`?

```suggestion
  psql -q -c "DROP TABLE IF EXISTS \"$1_diff_source\";"
  psql -q -c "ALTER TABLE \"$1\" RENAME TO \"$1_diff_source\";"
```